docs/3.0-migration-guide.d/remove_supp_for_extensions_in_pre-v3_X_509_certs.md

```diff
@@ -0,0 +1,11 @@
+Remove the `MBEDTLS_X509_ALLOW_EXTENSIONS_NON_V3` option
+--
+
+This change does not affect users who are working with current V3 X.509
+certificates.
+
+This change makes the pre-V3 X.509 certificates both with or without optional
+extensions obsolete.
```

Comment:
I don't think that's correct: the code change I see is only about extension, I don't think we're currently rejecting pre-v3 certificates in general. Suggested text: "Extensions were added in v3 of the X.509 specification, so pre-v3 certificates containing extensions were never compliant. Mbed TLS now rejects them with a parsing error in all configurations, as it did previously in the default configuration."

Reply:
Done